src/app/games/checkers/common/AbstractCheckersRules.ts

```diff
@@ -0,0 +1,517 @@
+import { Coord, CoordFailure } from 'src/app/jscaip/Coord';
+import { Ordinal } from 'src/app/jscaip/Ordinal';
+import { GameNode } from 'src/app/jscaip/AI/GameNode';
+import { Player } from 'src/app/jscaip/Player';
+import { ConfigurableRules } from 'src/app/jscaip/Rules';
+import { RulesFailure } from 'src/app/jscaip/RulesFailure';
+import { ArrayUtils, MGPFallible, MGPOptional, MGPValidation } from '@everyboard/lib';
+import { CheckersMove } from './CheckersMove';
+import { CheckersFailure } from './CheckersFailure';
+import { CheckersPiece, CheckersStack, CheckersState } from './CheckersState';
+import { GameStatus } from 'src/app/jscaip/GameStatus';
+import { TableUtils } from 'src/app/jscaip/TableUtils';
+import { CoordSet } from 'src/app/jscaip/CoordSet';
+import { Localized } from 'src/app/utils/LocaleUtils';
+
+export type CheckersConfig = {
+    playerRows: number;
+    emptyRows: number;
+    width: number;
+    stackPiece: boolean;
+    maximalCapture: boolean;
+    simplePieceCanCaptureBackwards: boolean;
+    promotedPiecesCanFly: boolean;
+    occupyEvenSquare: boolean;
+}
+
+export class CheckersLocalizable {
+
+    public static readonly STACK_PIECE: Localized = () => $localize`Stack piece instead of capturing it`;
```

Comment:
pluraliser ? "Empiler les pièces plutôt que les capturer" (une vérité vraie toute la partie) vs. "Empile la pièce plutôt que la capturer" (une formulation qui parle d'UNE pièce uniquement, pas d'une règle valide toute une partie)

Reply:
hein ?

Reply:
`Stack pieces instead of capturing them`

Reply:
done !